duckietown/Software
# Liam ground projection

Could probably still use a little cleanup but worked locally for lane following as well at least as the CPP version. 

To test:
make sure to pull, make build-catkin-clean, and make build-catkin-parallel 
on your robot and your laptop.

Then on robot run
$ make demo-lane-following

On laptop point ROS to the robot and open RVIZ and subscribe to the MarkerArrays (there are two - one published by duckiebot_visualizer and one published by lane_filter. 
With your robot in a lane you should see the proper colored lines projected onto the ground plane and the green arrow should be pointing in the direction that the robot is pointing roughly.
